lib/search/ui/widgets/search_bar.dart

```diff
@@ -0,0 +1,190 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:fluttercon/common/utils/router.dart';
+import 'package:fluttercon/common/widgets/resolved_image.dart';
+import 'package:fluttercon/core/di/injectable.dart';
+import 'package:fluttercon/core/theme/theme_colors.dart';
+import 'package:fluttercon/search/cubit/search_cubit.dart';
+import 'package:fluttercon/search/cubit/search_state.dart';
+import 'package:fluttercon/search/models/search_result.dart';
+import 'package:go_router/go_router.dart';
+
+class SearchBarWidget extends StatelessWidget {
+  const SearchBarWidget({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider(
```

Comment:
Once you have registered the bloc in `bootstrap.dart`, you don't need this `BlocProvider` here.

Reply:
fixed